diff -ru jsoncpp-0.10.5/CMakeLists.txt jsoncpp-0.10.5.patched/CMakeLists.txt
--- jsoncpp-0.10.5/CMakeLists.txt	2015-07-23 07:32:47.000000000 +0200
+++ jsoncpp-0.10.5.patched/CMakeLists.txt	2016-02-16 19:10:50.731887585 +0100
@@ -97,10 +97,10 @@
 
 if (CMAKE_CXX_COMPILER_ID MATCHES "Clang")
   # using regular Clang or AppleClang
-  set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++11 -Werror -Wall -Wshadow -Wshorten-64-to-32")
+  set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++11 -Wall -Wshadow -Wshorten-64-to-32")
 elseif ("${CMAKE_CXX_COMPILER_ID}" STREQUAL "GNU")
   # using GCC
-  set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Werror -Wall -Wshadow -Wextra -pedantic -Wno-long-long")
+  set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wall -Wshadow -Wextra -pedantic -Wno-long-long")
   # not yet ready for -Wconversion
 endif()
 
